### PR TITLE
 RHIDP-1294 feat(arches): add supported arch list -...

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

feat(arches): add supported arch list - RHIDP-1294

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
RHIDP-1294

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: release-notes/con-release-notes-overview.adoc
Comment (linfraze) [inline release-notes/con-release-notes-overview.adoc:20]:
```suggestion
* OpenShift Container Platform 4.12, 4.13, 4.14, 4.15
```

File: release-notes/con-release-notes-overview.adoc
Comment (linfraze) [inline release-notes/con-release-notes-overview.adoc:32]: I trust that you know that this is true and there are no errors here, but another SME or QE should review/confirm this before merging.

**Review by linfraze — COMMENTED** [review]
Peer review complete. Please also get a SME/QE reviewer to confirm technical accuracy.

File: release-notes/con-release-notes-overview.adoc
Comment (nickboldt) [reply release-notes/con-release-notes-overview.adoc:32]: Well, you can look up the options from https://catalog.redhat.com/software/containers/rhdh/rhdh-hub-rhel9/645bd4c15c00598369c31aba?architecture=amd64&image=660ebff0036e65b1bc611db4 if you don't believe me :rofl: 

image

Comment (nickboldt) [reply release-notes/con-release-notes-overview.adoc:32]: this is how we documented arch support for Dev Spaces, which was available on 3 arches. https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.12/html-single/release_notes_and_known_issues/index#supported-platforms

Comment (linfraze) [reply release-notes/con-release-notes-overview.adoc:32]: @nickboldt Cool, thanks. I didn't have the knowledge to confirm the technical accuracy of this change and wasn't sure how/where to check for confirmation and didn't want to merge anything without a tech check.

Comment (briandooley) [reply release-notes/con-release-notes-overview.adoc:32]: @nickboldt From now on, we need to get an SME or QE reviewer for all PRs merging to the docs repo. In the interests of time, if you could find someone to do the review, that would be a great help. Otherwise, the docs team can find someone.

Comment (nickboldt) [reply release-notes/con-release-notes-overview.adoc:32]: I thought I was an SME here, given I do the builds and cited my sources, go ahead and conscript someone in QE or another SME to click on the above links to verify my claim that we only build on one architecture. In this case it feels like you're making work for others, but I understand that process-wise it's uncool to have the PR come from the SME who does the SME review.

On the other hand, I've provided you with the doc PR that a writer would have otherwise had to submit to ME to review, so I feel like ultimately we're getting to the same place?

Comment (linfraze) [reply release-notes/con-release-notes-overview.adoc:32]: @nickboldt I totally hear you. But as a docs expert, I am still required to have my work checked by another docs expert in the form of a peer review, even for minor typo fixes. This makes me grumble, but I accept that those are the rules of contributing to this repo. Maybe the rules themselves need to be revisited. But in the meantime, Brian is trying to standardize the review process for all docs repo contributions. Otherwise the docs team is held to rules that other contributors are exempt from, despite being the primary maintainers of the repo.

**Review by linfraze — APPROVED** [review]
thanks for the source links - lgtm

**Pull request merged**
